ci-operator/step-registry/openshift-qe/installer/aws/sdn/OWNERS

```diff
@@ -0,0 +1 @@
+../../OWNERS
```

Comment:
this owners file should refernce back to the OWNERS file here

Think it needs 3 "../"